### DISCUSSION TITLE
How to make exception handling better compliant to spec

### DISCUSSION THREAD

**Discussion by n2ygk** [django-json-api/django-rest-framework-json-api#924] [Ideas]
Split #416 out into exception handling improvements which return useful jsonapi 1.0 [error objects](http://jsonapi.org/format/#error-objects)

**Comment by sliverc** [comment]
Converted this issue to a discussion so we can talk about what are the actually issue with the current exception handling and and out of this can hopefully extract single tasks/issue how to work on it.
